app/templating/summary/question.py

```diff
@@ -24,13 +24,20 @@ def _build_answers(cls, block_id, question_schema, answer_schema, answers):
         summary_answers = []
         answers_iterator = iter(answer_schema)
         for answer_schema in answers_iterator:
-            answer = cls._build_answer(question_schema, answer_schema, answers, answers_iterator)
-            summary_answers.append(Answer(block_id, answer_schema, answer))
+            if question_schema['type'] == 'RepeatingAnswer':
+                for k, v in answers.items():
```

Comment:
If we don't use `v` I think the pythonic way of ignoring it is with an underscore `k, _`